pkg/util/labels/labels.go

```diff
@@ -20,6 +20,19 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// Contains report whether labels are within l
```

Comment:
ping @liggitt for reviewing this logic

Reply:
seems this folder is readonly, but the helpers in apimachinery for this case are not useful, should I move this helper to the endpoints utils?

Reply:
An alternative to this function might be to update `setServiceLabels` to simply return true if it changed anything.

Reply:
I added it in endpointslices utils as an independent function, look much cleaner to me.